src/service.rs

```diff
@@ -146,7 +143,7 @@ impl<Types: NodeType> GlobalState<Types> {
         bootstrapped_builder_state_id: VidCommitment,
         bootstrapped_view_num: Types::Time,
         last_garbage_collected_view_num: Types::Time,
-        buffer_view_num_count: u64,
+        _buffer_view_num_count: u64,
```

Comment:
We should just get rid of this if we're getting rid of it, so we can clean up the env vars on the sequencer side

Reply:
Yes. I have another set of changes that will clean up some dangling bits, including this one, but requires corresponding changes in the sequencer repo. I wanted to test with no external changes required first.